firebase_admin/remote_config.py

```diff
@@ -197,7 +197,7 @@ def get_float(self, key):
         return self.get_value(key).as_float()
 
     def get_value(self, key):
```

Comment:
Could we add type hints for all the functions where possible, please?
We are trying to add type hints to any new code. For example, see: https://github.com/firebase/firebase-admin-python/blob/43ab91e22d02e5d07dfcc77e6943f4d1251c291e/firebase_admin/app_check.py#L73

Reply:
Just noticed that you have added typehints to return types in most of the public functions. Thanks!